ext/curl/tests/curl_setopt_ssl.phpt

```diff
@@ -61,7 +61,22 @@ if ($serverCert === false
 $port = 14430;
 
 // set up local server
-$cmd = "openssl s_server -key $serverKeyPath -cert $serverCertPath -accept $port -www -CAfile $clientCertPath -verify_return_error -Verify 1";
+$cmd = [
```

Comment:
I am also playing arround with this kind of array-type arg passing to `proc_open`.

I wonder how its possible to make stderr to stdout redirection work in this situation. usually I would use `'2>&1'` but it seems in the array-type notation, this arg gets escaped?

Reply:
The command is executed as is by the operating system, there is no shell , ergo no redirection. 
proc_open implements the redirect option in descriptorspec (which is unfortunately not documented) see the general_functions/proc_open_redirect.phpt test for examples on how to do the 2>&1

Reply:
thank you for taking the time to answer my silly questions. really appreciate it.